benchmarks/conv_py.py

```diff
@@ -0,0 +1,40 @@
+import jax
+import dex
+from dex.interop import jax as djax
+
+import time
+import timeit
+
+def bench_python(f, loops=None):
+  """Return average runtime of `f` in seconds and number of iterations used."""
+  if loops is None:
+    f()
+    s = time.perf_counter()
+    f()
+    e = time.perf_counter()
+    duration = e - s
+    loops = max(4, int(2 / duration)) # aim for 2s
+  return (timeit.timeit(f, number=loops, globals=globals()) / loops, loops)
+
+
+def block_until_ready(array):
+  # A block_until_ready variant that works on numpy arrays too
```

Comment:
Why do you need it to work on NumPy arrays? A JAX primitive should always return JAX arrays I think?

Reply:
It should, but empirically `dex_apply` turns out not to.  That's probably a bug we should probably deal with, but I'd rather do that in a separate change.

Reply:
Ok. In that case let's just put an assert that the dex function returns a NumPy function somewhere and remove this `block_until_ready`. I don't know what is the cost of Python exception handling, but we shouldn't be paying it in the hot part of our benchmark.

Reply:
Done.